lib/usd/translators/strokeWriter.h

```diff
@@ -16,22 +16,17 @@
 #ifndef PXRUSDTRANSLATORS_STROKE_WRITER_H
 #define PXRUSDTRANSLATORS_STROKE_WRITER_H
 
-/// \file pxrUsdTranslators/strokeWriter.h
```

Comment:
Would be good to leave this Doxygen tag if we can. You can truncate it to just this:

```
/// \file
```

Reply:
See c6eecc5